packages/terra-clinical-header/CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ ChangeLog
 
 Unreleased
 ----------
+### Added
+* Added orion-fusion-theme style values
+
+### Changes
```

Comment:
```suggestion
### Changed
```

https://engineering.cerner.com/terra-ui/contributing/terra-ui/developer-workflow#updating-the-changelog